<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewProfileToolboxesDialog</class>
 <widget class="QDialog" name="ViewProfileToolboxesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure View Profiles</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Select the windows, that should be visible, when the different profiles are active.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="editGroup">
     <property name="title">
      <string>&amp;Edit Profile</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="epltCheckBox">
        <property name="text">
         <string>Left Toolbox</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="eprtCheckBox">
        <property name="text">
         <string>Right Toolbox</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ephtCheckBox">
        <property name="text">
         <string>Horizontal Toolbox</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="debugGroup">
     <property name="title">
      <string>&amp;Debug Profile</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="dpltCheckBox">
        <property name="text">
         <string>Left Toolbox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dprtCheckBox">
        <property name="text">
         <string>Right Toolbox</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dphtCheckBox">
        <property name="text">
         <string>Horizontal Toolbox</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>epltCheckBox</tabstop>
  <tabstop>eprtCheckBox</tabstop>
  <tabstop>ephtCheckBox</tabstop>
  <tabstop>dpltCheckBox</tabstop>
  <tabstop>dprtCheckBox</tabstop>
  <tabstop>dphtCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ViewProfileToolboxesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ViewProfileToolboxesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
